### ISSUE TITLE
Context + one_of() issue

### ISSUE THREAD

**Issue by forslund** [MycroftAI/adapt#86]
When a keyword declared with one_of is provided via context the phrase isn't populated and is instead `None`:
```json
{
    "intent_type": "WeatherIntent",
    "WeatherKeyword": "weather",
    "LocationContext": null,
    "target": null,
    "confidence": 1.0
}
```
above LocationContext is null/None

When specified as required it works as expected:
```json
{
    "intent_type": "WeatherIntent",
    "WeatherKeyword": "weather",
    "LocationContext": "lizard",
    "target": null,
    "confidence": 0.375
}
```
here LocationContext is set to the value from context.

For a minimal example see https://gist.github.com/forslund/d98c98816dc90600aed7ad62bfd0f223

**Comment by penrods** [comment]
Once this is addressed, remove the Mycroft-Core workaround:
https://github.com/MycroftAI/mycroft-core/pull/1789

**Comment by forslund** [comment]
This should be fixed now through #103, closing issue
